studio.assetmanager.ams.yaml

```diff
@@ -0,0 +1,77 @@
+app-id: studio.assetmanager.ams
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: ams
+finish-args:
+  # X11 performance
+  - --share=ipc
+  # Access to network
+  - --share=network
+  # Access to X11
+  - --socket=x11
+  # Access to wayland
+  - --socket=wayland
+  # Audio Access
+  - --socket=pulseaudio
+  # Access to all files
+  - --filesystem=host
+  # Access to OpenGL
+  - --device=dri
+  # Use same mouse cursors as host
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --talk-name=org.freedesktop.secrets
+
+rename-desktop-file: asset-manager-studio.desktop
+rename-icon: asset-manager-studio
+modules:
+  - name: ams-desktop
+    buildsystem: simple
+    build-commands:
+      # Extract .deb & cleanup
+      - ar -x asset-manager-studio*.deb
+      - rm -f asset-manager-studio*.deb
+      - tar -xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+
+      # Copy application to Flatpack destination, and then clean it up
+      - cp --preserve --recursive opt/* usr/* $FLATPAK_DEST
+      - rm -rf opt usr
+
+      # Install our two data files
+      - install ams.sh $FLATPAK_DEST/bin/ams
+      - install -Dm0644 "${FLATPAK_ID}.metainfo.xml" "${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml"
+
+      # update Desktop file
+      - desktop-file-edit --set-key=Exec --set-value="ams %U" $FLATPAK_DEST/share/applications/asset-manager-studio.desktop
+      - desktop-file-edit --set-key=Icon --set-value="${FLATPAK_ID}" "${FLATPAK_DEST}/share/applications/asset-manager-studio.desktop"
+      - desktop-file-edit --set-key=X-Flatpak-RenamedFrom --set-value='asset-manager-studio.desktop;' "${FLATPAK_DEST}/share/applications/asset-manager-studio.desktop"
```

Comment:
```suggestion
```

you never need to do that. What bit of the documentation said so (so we fix it)

Reply:
Copied it from Signal's repo; I'd rather have overkill and have everything work then wonder "Why" they did something and have weird issues.   So I copied anything that I wasn't sure why they were using.  :D

https://github.com/flathub/org.signal.Signal/blob/0e46425c4bafd2076351bd6d1244a11d38fc1e2d/org.signal.Signal.yaml#L82

Reply:
once again two wrong don't make a right. Signal is not the documentation.

Reply:
Unfortunately the documentation has lots of issues (much outdated it seems) when dealing with Electron apps.  Going to actual published electron flatpak apps was the only way I made it work using Flatpak.